### DISCUSSION TITLE
Kiali not discover gateways when deploying Istio into multiple namespaces

### DISCUSSION THREAD

**Discussion by saddamAlTamimi** [kiali/kiali#5242] [General]
Hello,

When I deployed Anthos Service mesh as per google recommendations into two namespaces,  I noticed gateways were not discovered by kiali and no traffic was showing on the dashboard.

I tried all. Kiali version over ASM  version > 1.11. till. 1.13 .

Any idea, since I am using Kiali.yaml directly  for  the deployments.

**Comment by lucasponce** [comment]
Hi @saddamAlTamimi,

 We don't have any known issues around, so, my first guess it's about the environment.

Perhaps you could check if you are installing a supported version according to your Istio installation:
https://kiali.io/docs/installation/installation-guide/prerequisites/#version-compatibility

Also for private clusters, this could be relevant:
https://kiali.io/docs/faq/general/#what-do-i-need-to-run-kiali-in-a-private-cluster

But if that doesn't help, perhaps some captures of what you see and/or logs may help to get some hints about the problem.

**Comment by jmazzitelli** [comment]
(converted this to a github discussion)

**Reply by saddamAlTamimi** [reply]
Hello @lucasponce,

Thanks for. reply.
I managed to have Kiali 1.49.0 which is compatible wit Istio "ASM v1.13" ad allowed ports for private cluster, eve wit these steps still Kiali dashboard sop issue wit discovery Istio gatewys. we used to deploy GitHub repository https://www.github.com/istio/istio/blob/master/samples/addons/kiali.yaml

Do I need. to change or. customize any. part to cover Istio namespaces ?!!
